docs/application/dotnet/guides/machine-learning/singleshot.md

```diff
@@ -0,0 +1,120 @@
+# Single Shot
+
+With `Tizen.MachineLearning.Inference.SingleShot` class, you can load the existing neural network model or your own specific model from storage and invoke it with a single instance of input data, finally get the inference output result.
+
+The main features of the Tizen.MachineLearning.Inference are:
+
+- Managing tensor information, which is the metadata: dimensions and types of tensors
+
+  You can [configure the input and output Tensor Information](#manage) such as its name, data type and dimension.
+
+- Loading a neural network model and configuring a runtime environment
+
+  You can [load the neural network model from storage and configure a runtime environment](#load).
+
+- Invoking the neural network model with input data
+
+  After setting up the SingleShot instance with its required information, you can [invoke the model with the input data and get the inference output result](#invoke).
+
+- Fetching the inference result after invoking
+
+  You can [fetch the inference result after invoking](#fetch).
+
+## Prerequisites
+
+To use Machine Learning Inference with your application:
+
+1. To use the methods and properties of the `Tizen.MachineLearning.Inference.SingleShot` class or its related classes such as `Tizen.MachineLearning.Inference.TensorsData` and `Tizen.MachineLearning.Inference.TensorsInfo`, include the [Tizen.MachineLearning.Inference]() namespace in your application:
+    ```C#
+    using Tizen.MachineLearning.Inference;
+    ```
+
+2. If the model file you want to use is located at `Media Storage` or `External Storage`,
+the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+    ```xml
+    <privileges>
+      <!--To access media storage-->
+      <privilege>http://tizen.org/privilege/mediastorage</privilege>
+
+      <!--To access, read, and write to the external storage-->
+      <privilege>http://tizen.org/privilege/externalstorage</privilege>
+    </privileges>
+    ```
+
+<a name="manage"></a>
```

Comment:
Hmm..
You mean I should use **id** instead of **name** as below. right? 
```html
<a id="manage"></a>
```

Reply:
@again4you 
In markdown file, the link for the titles with heading tag \# is automatically created.

In case of \## Managing tensor information,
you'd possible to access this point with "#managing-tensor-information"

![image](https://user-images.githubusercontent.com/9958374/67444374-b9165100-f643-11e9-9b07-438662372713.png)

So @Reni-90's comment means,
not to use another way to set the link, and use the link tag generated automatically.

Anyway, using "\<a name="manage"></a>" is an optional way, and it is also possible for some cases.